cicecore/cicedyn/infrastructure/ice_restoring.F90

```diff
@@ -594,13 +677,20 @@ subroutine ice_HaloRestore
 !-----------------------------------------------------------------------
 
       ! for now, use same restoring constant as for SST
-      if (trestore == 0) then
-         trest = dt          ! use data instantaneously
-      else
-         trest = real(trestore,kind=dbl_kind) * secday ! seconds
-      endif
-      ctime = dt/trest
-
+!       if (trestore == 0) then
+!          trest = dt          ! use data instantaneously
+!       else
+          trest_ice = real(.5,kind=dbl_kind) * 3600!secday ! seconds
```

Comment:
@SmithJos13 It seems this will affect other cases/configurations. Please make those changes by thinking about other cases. The PR needs to pass other CICE tests under UFS WM (not Coastal).